### PR TITLE
[ACM-14264]: Backport proxy prereq to HCP docs

### PR DESCRIPTION
**Pull request by lahinson**
Related Jira: https://issues.redhat.com/browse/ACM-14264

After the wording is approved, it needs to also be backported to the following versions:
- 2.10
- 2.9

### CODE REVIEW

File: clusters/hosted_control_planes/kubevirt_intro.adoc [hunk 1]
```diff
@@ -57,6 +57,8 @@ oc patch storageclass ocs-storagecluster-ceph-rbd -p '{"metadata": {"annotations
 oc get managedclusters local-cluster
 ----
 
+- If you are using a proxy, the `NO_PROXY` environment variable in {ocp-virt-short} must have an IP address of at least 172.20.0.1.
```
Comment (enxebre) [inline clusters/hosted_control_planes/kubevirt_intro.adoc]: This apply to all platforms.
Also this won't be necessary after https://github.com/openshift/hypershift/pull/4693 is shipped by mce
I think we'll also need to document how to configure proxy in general, e.g. run hcp create cluster --render, edit hc.spec.config.proxy...

Comment (enxebre) [reply clusters/hosted_control_planes/kubevirt_intro.adoc]: Also it doesn't really must, it's a config choice. I'd say something like "if you are using proxy and you want traffic from pods to the kube api server to not use the proxy you need to add our default kas advertised address 172.20.0.1 to the no_proxy list.

**Review by swopebe — APPROVED** [review]
looks great

**Pull request merged**